Util.php

```diff
@@ -20,4 +16,33 @@ public static function cleanDomain(string $domain): string
 
         return $domain;
     }
+
+    /**
+     * Retrieve messages from translator bag.
+     *
+     * @param TranslatorBagInterface $translatorBag
+     * @param string $locale
+     * @param string $domain
+     * @return array 2D array, where first-level keys are domain without suffixes, and domain with '+intl-icu' suffix. Second level keys are translation keys.
```

Comment:
`@return array<string, array<string, string>>` would allow IDEs and static analyzers to known how this array looks like instead of having only a human-readable explanation (which is still needed to understand what the strings are about)

Reply:
Thank you! I've incorporated the more precise type annotation.